src/main/java/com/dungeoncrawler/Javiarenka/character/Hero.java

```diff
@@ -18,20 +22,23 @@
     private String armorName;
     private String className;
     private int money;
+    private ArrayList<Skill> skills;
```

Comment:
Deklarować listę możesz bez używania "ArrayList". Dopiero przy inicjalizacji musisz wskazać że to ArrayLista, na przykład:
`List<String> someList = new ArrayList<>()`

Reply:
Racja, zapomniałem o tym